docs/source/identity/identity.md

```diff
@@ -254,7 +254,7 @@ trusted members of a blockchain network. That's where a Membership Service Provi
 (MSP) comes into play --- **it identifies the parties who are the members of a
 given organization in the blockchain network**.
 
-To learn more about membership, check out the conceptual documentation on [MSPs](../membership/membership.html).
+To learn more about membership, check out the conceptual documentation on [MSPs](../membership/membership.md).
```

Comment:
I'm pretty sure the old link is the correct one. This page is rendered [here](https://hyperledger-fabric.readthedocs.io/en/release-2.0/identity/identity.html) and the link becomes https://hyperledger-fabric.readthedocs.io/en/release-2.0/membership/membership.html.

If you're attempting to follow links in within the repository on GitHub, they won't work.

Reply:
Hi I'm reading the documentation to see if it is a good behaviour. If we can use, another configuration, to use .md path links on github and transform to .html for exhibit on the page, something like this.

Reply:
Looking in the [convention](https://github.com/hyperledger/fabric/tree/b3ce17d4a7c02f5f7f559ea2d289470d57378f91/docs#conventions), we can see this note: 

> Relative links are unfortunately not working on github when browsing through a RST file.

But [relative link](https://github.com/hyperledger/fabric/blob/b3ce17d4a7c02f5f7f559ea2d289470d57378f91/docs/source/getting_started.rst) works for another pages not inside a same page (anchors), and the project have some files in format .md - this one for example **docs/source/identity/identity.md**, why? This is some workaround for the problem with .rst files?

Good when I can I test this on my local machine, But if someone have the response say me.

thx

Reply:
Matt is correct. We use a tool called tox to render our webpages a third-party tool to convert .md and .rst files into .html files. ReadTheDoc utilizes our tox configuration to picks the pages and render them. So when you are reading the doc in the `docs/source` directory, its not meant to be read locally, it has to be built using the tox engine and then you need to view it in its native `.html` format, not its `.md` or `.rst` format. So opening it in your IDE or browser without rendering it with tox is not supported.